src/applications/disability-benefits/new-686/config/generic-schema.js

```diff
@@ -27,4 +27,31 @@ export const genericSchemas = {
     enum: states50AndDC.map(state => state.value),
     default: states50AndDC.map(state => state.label),
   },
+  fullName: {
+    type: 'object',
+    properties: {
+      first: {
+        type: 'string',
+        minLength: 1,
+        maxLength: 30,
+        pattern: '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*0-9]+$',
+      },
+      middle: {
+        type: 'string',
+        maxLength: 20,
+        pattern: '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*0-9]+$',
+      },
+      last: {
+        type: 'string',
+        minLength: 1,
+        maxLength: 30,
+        pattern: '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*0-9]+$',
+      },
+    },
+    required: ['first', 'last'],
+  },
+  date: {
+    type: 'string',
+    pattern: '^(\\d{4}|XXXX)-(0[1-9]|1[0-2]|XX)-(0[1-9]|[1-2][0-9]|3[0-1]|XX)$',
```

Comment:
Would one of the common date schemas work here?

Reply:
Yes, definitely. I think our plan was to take care of that type of thing when we migrate our schemas over to the `vets-json-schema` repo.